src/main/java/uk/gov/hmcts/reform/civil/enums/dq/GAJudgeDecisionOption.java

```diff
@@ -6,7 +6,8 @@
 @Getter
 @RequiredArgsConstructor
 public enum GAJudgeDecisionOption {
-    MAKE_AN_ORDER("Make an order"),
+    MAKE_AN_ORDER("Make an order without a hearing"),
```

Comment:
there needs to be test changes done on ga ccd . . then raise a test PR in ga ccd pointing to this image . .thanks

Reply:
Done